releasenotes/notes/move-snmp-listener-config-06a03c25b9f19b55.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    This moves the configuration for `snmp_listener` under `network_devices`. While the old configuration will continue to work, if both configurations are present, the new one will take priority.
```

Comment:
**🥜 nitpick:** release notes are meant to inform users about what changed in a release ([see example](https://github.com/DataDog/datadog-agent/releases/tag/7.52.0)) so having `This` (referring to the PR) feels a bit weird imho.

I believe this would be good for a PR description but for a release note I think something like `"snmp_listener" configuration now lives under "network_devices"` would be a better wording (but you got an approval from the documentation team so feel free to ignore)‏

Reply:
totally agree with you
https://github.com/DataDog/datadog-agent/pull/24644/commits/185c786da6905da7596ac5f5e856f1d9b11f7080